src/fonts/inter.scss

```diff
@@ -82,6 +82,18 @@ Usage:
   src: url('woff2/Inter-roman.var.woff2') format('woff2');
 }
 
+/* Minimal font file with limited glyphs and layout features (only kern).
+   Should be enough for most en-US pages. */
+@font-face {
+  font-family: 'Inter var';
+  font-style: normal;
+  font-weight: 100 900;
```

Comment:
At one point during our discussion yesterday you reduced the number of weights too?

Reply:
I haven't been able to figure out how to do this yet (or if this is even possible) - I thought I had it at some point but upon close inspection it didn't do anything.

Reply:
Filed https://github.com/mozilla/addons-frontend/issues/10500 for follow-up